src/main/java/org/gridsuite/modification/server/NetworkModificationService.java

```diff
@@ -49,20 +54,26 @@ void changeSwitchState(UUID networkUuid, String switchId, String open) {
         sw.setOpen(Boolean.parseBoolean(open));
 
         networkStoreService.flush(network);
+
+        return listener.getModifications();
     }
 
-    boolean applyGroovyScript(UUID networkUuid, String groovyScript) {
+    public Pair<Boolean, Set<String>> applyGroovyScript(UUID networkUuid, String groovyScript) {
```

Comment:
Use a dedicated object for this to have a nicer API

Reply:
Done